src/validation_workflow/validation.py

```diff
@@ -0,0 +1,25 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+
+import argparse
+from abc import ABC, abstractmethod
+
+
+class Validation(ABC):
+    url = "https://artifacts.opensearch.org/releases/bundle/"
+
+    def __init__(self) -> None:
+        parser = argparse.ArgumentParser(description="Download Artifacts.")
+        parser.add_argument("version", help="Enter Version.")
```

Comment:
Why we have another argument parser in this class? Should all be handled in the `ValidationArgs()` and passing that `args` assigned in `run_validation` within all classes?

Reply:
I haven't called the base class in my workflow by creating an object,So I have left it to default version. I will pass all the arguments from ValidationArgs() to __init__ if the base class object is utilized in further development of the code or can set it to default without passing and arguments and without declaring any new variables.